features/Websocket.feature

```diff
@@ -312,38 +312,140 @@ Feature: Test websocket API
     Then I am able to get the user "user2-id" matching {"_id":"#prefix#user2-id","_source":{"profile":{"_id":"#prefix#profile2"}}}
     Then I search for {"regexp":{"_uid":"users.#prefix#.*"}} and find 2 users
     Then I delete the user "user2-id"
-    Then I search for {"regexp":{"_uid":"users.#prefix#.*"}} and find 1 users matching {"_id":"#prefix#user1-id","_source":{"name":{"first":"David","last":"Bowie"}}}
-    When I log in as user1-id:testpwd expiring in 1h
-    Then I am getting the current user, which matches {"_id":"#prefix#user1-id","_source":{"profile":{"_id":"admin"}}}
+    Then I search for {"regexp":{"_uid":"users.#prefix#.*"}} and find 1 users matching {"_id":"#prefix#useradmin-id","_source":{"name":{"first":"David","last":"Bowie"}}}
+    When I log in as useradmin-id:testpwd expiring in 1h
+    Then I am getting the current user, which matches {"_id":"#prefix#useradmin-id","_source":{"profile":{"_id":"admin"}}}
     Then I log out
     Then I am getting the current user, which matches {"_id":-1,"_source":{"profile":{"_id":"anonymous"}}}
 
   @usingWebsocket @cleanSecurity
   Scenario: user updateSelf
-    When I create a new user "user1" with id "user1-id"
-    Then I am able to get the user "user1-id" matching {"_id":"#prefix#user1-id","_source":{"profile":{"_id":"admin", "_source": {"roles":[{"_id":"admin"}]}}}}
-    When I log in as user1-id:testpwd expiring in 1h
-    Then I am getting the current user, which matches {"_id":"#prefix#user1-id","_source":{"profile":{"_id":"admin"}}}
+    When I create a new user "useradmin" with id "useradmin-id"
+    Then I am able to get the user "useradmin-id" matching {"_id":"#prefix#useradmin-id","_source":{"profile":{"_id":"admin", "_source": {"roles":[{"_id":"admin"}]}}}}
+    When I log in as useradmin-id:testpwd expiring in 1h
+    Then I am getting the current user, which matches {"_id":"#prefix#useradmin-id","_source":{"profile":{"_id":"admin"}}}
     Then I update current user with data {"foo":"bar"}
-    Then I am getting the current user, which matches {"_id":"#prefix#user1-id","_source":{"profile":{"_id":"admin"},"foo":"bar"}}
+    Then I am getting the current user, which matches {"_id":"#prefix#useradmin-id","_source":{"profile":{"_id":"admin"},"foo":"bar"}}
     Then I log out
     Then I am getting the current user, which matches {"_id":-1,"_source":{"profile":{"_id":"anonymous"}}}
 
   @usingWebsocket @cleanSecurity @unsubscribe
   Scenario: token expiration
     Given A room subscription listening to "lastName" having value "Hopper"
-    Given I create a user "user1" with id "user1-id"
+    Given I create a user "useradmin" with id "user1-id"
     When I log in as user1-id:testpwd expiring in 1s
     Then I wait 1s
     And I should receive a "jwtTokenExpired" notification
 
+  @usingWebsocket @cleanSecurity @unsubscribe
```

Comment:
was just a copy-past from the former line, but yes, you're right, @unsubscribe  is not needed here.